### ISSUE TITLE
Update made my dialer disappear

### ISSUE THREAD

**Issue by barudaret** [OpenLauncherTeam/openlauncher#532]
#### General Information

* **OpenLauncher Version:** 0.7.2?
* **Android Version:** 9?
* **Custom ROM:** None (Stock MIUI)
* **Install Source:** F-Droid

#### Description
After the latest update, the dialer disappeared.  If I witch to the stock MIUI launcher, the dialer is there, but not in openlauncher.  To make matters worse, I foolishly granted this app sys-app status recently.  So I can't downgrade it or uninstall it.  Can't even clear app data, apparently.  I'm at the mercy of whenever the next update comes out.  I can't find anything in the settings to relinquish sysapp status, but I don't expect to find anything like that anyway. 

#### Log

This is only needed for crashes or other issues you notice in your system logs from OpenLauncher.

**Comment by phanirithvij** [comment]
This might help you https://play.google.com/store/apps/details?id=com.jumobile.manager.systemapp&hl=en

**Comment by barudaret** [comment]
Thanks @phanirithvij, I don't have root though.  Last I heard, you have to wait something like two weeks to get approval from xiaomi before you can root/install ROMs, etc.  So I may as well wait for the next openlauncher update

**Comment by Catfriend1** [comment]
I have this problem, too.
My dialer on xiaomi mi8 miui 11 disappeared with the update to Fdroid 0.7.2 version.

Furthermore, my Syncthing Camera App Icon disappeared at the same time. As the cam icon was a second launcher intent for the Syncthing-Fork app, I suspect Openlauncher to have problems with secondary launcher intents?!

**Comment by dkanada** [comment]
It's 100% to blame, but I don't have the time to figure out why at the moment. It's also the only thing I added to 0.7.2 so you can just revert to the last version for now and it will work fine.

**Comment by hobleyd** [comment]
The issue is something to do with apps which share the same executable but are named differently, I think. Contacts is the same app as the Dialler, but named differently. If I remove the duplicate check, I get two Contacts apps.

**Comment by dkanada** [comment]
Yeah, there's also some kind of issue with work profile apps picking a different activity than normal, which means icon packs don't recognize them. This was quite annoying as well.

**Comment by Catfriend1** [comment]
Fyi: export, downgrade to 0.7.1-fdroid and import fixed the problem for me. So a new release would be appreciated but is not urgently required.
